src/lib/Asset.ts

```diff
@@ -297,6 +297,13 @@ class Asset {
     }
   }
 
+  /**
+   @notice Gets the contract
```

Comment:
@byterose How do I write the docs for this? I was thinking 
```suggestion
   @notice Gets the contract
   @returns Contract as an ExpiringMultiPartyEthers or LongShortPairEthers.
```

That seem descriptive enough?

Reply:
I would suggest to write it like this
```   
@notice Gets the contract
@returns Contract of type ExpiringMultiPartyEthers or LongShortPairEthers.
```